app/Services/PlanetService.php

```diff
@@ -353,20 +350,17 @@ public function deductResources(Resources $resources, bool $save_planet = true):
      */
     public function hasResources(Resources $resources): bool
     {
-        if (!empty($resources->metal->get())) {
-            if ($this->metal()->get() < $resources->metal->get()) {
-                return false;
-            }
+        if ( !empty( $resources->metal->get() ) && $this->metal()->get() < $resources->metal->get() )
+        {
+            return false;
         }
-        if (!empty($resources->crystal->get())) {
-            if ($this->crystal()->get() < $resources->crystal->get()) {
-                return false;
-            }
+        if ( !empty( $resources->crystal->get() ) && $this->crystal()->get() < $resources->crystal->get() )
+        {
+            return false;
         }
-        if (!empty($resources->deuterium->get())) {
-            if ($this->deuterium()->get() < $resources->deuterium->get()) {
-                return false;
-            }
+        if ( !empty( $resources->deuterium->get() ) && $this->deuterium()->get() < $resources->deuterium->get() )
+        {
+            return false;
```

Comment:
Please remove spaces surrounding if() and !empty() to adhere to PSR-2 standards.